### PR TITLE
test(labware-creator): more test refactoring

### PR DESCRIPTION
**Pull request by IanLondon**
# Overview

This PR updates existing tests for the different sections in LC.

# Changelog

- test(labware-creator): more test refactoring

# Review requests

- [ ] Code review

# Risk assessment

Low, test refactors in LC only

### CODE REVIEW

File: labware-library/src/labware-creator/components/__tests__/sections/WellBottomAndDepth.test.tsx [hunk 1]
```diff
@@ -78,56 +36,53 @@ describe('WellBottomAndDepth', () => {
...
+    screen.getAllByRole('radio', { name: /v-bottom/i })
+
+    screen.getByRole('textbox', { name: /depth/i }) // TODO IMMEDIATELY this should work after Sarah's PR is merged & this is rebased
   })
 
-  it('should render tubes when tubeRack is selected', () => {
-    formikConfig.initialValues.labwareType = 'tubeRack'
+  it('should render tubes when labware that should displayAsTube is selected', () => {
     when(displayAsTubeMock)
-      .calledWith(formikConfig.initialValues)
```
Comment (shlokamin) [inline labware-library/src/labware-creator/components/__tests__/sections/WellBottomAndDepth.test.tsx:95]: @mcous  and i had [a long chat ](https://github.com/Opentrons/opentrons/pull/7737#discussion_r623354742)about `expect` vs `expectCalledWith` (i.e. mocks vs stubs).  i documented the outcome that we prefer mocks over stubs in the [frontend cookbook decision log](https://coda.io/d/Front-end-Cookbook_dw3eEFMgHPr/Decision-Log_suv6v#Decisions_tuWnn/r7)

i dont think we need to be too pedantic about this, just thought i'd comment. what do u think?

Comment (IanLondon) [reply labware-library/src/labware-creator/components/__tests__/sections/WellBottomAndDepth.test.tsx:95]: I'm kinda confused by the discussion and how it relates to that conclusion. The decision log says mocks would do `expectCalledWith`, but @mcous is arguing against `expectCalledWith` and it seems like you meet him there, so I don't get how you two concluded mocks over stubs it seems like you both said the opposite?

**Review by Kadee80 — APPROVED** [review]
🧸 LGTM - also doesnt look like it will interfere with @smb2268's Total Height PR

File: labware-library/src/labware-creator/components/__tests__/sections/WellBottomAndDepth.test.tsx (hunk 1 above)
Comment (shlokamin) [reply labware-library/src/labware-creator/components/__tests__/sections/WellBottomAndDepth.test.tsx:95]: ARGH I am so sorry @IanLondon I have been using the terms backwards, which is super confusing, 10000% my bad. 

The decision is to prefer STUBS (pre canned answers, making no assertions when called in an unexpected way). Super sorry for using the wrong term, must have been really frustrating trying to understand what we meant :(

Comment (shlokamin) [reply labware-library/src/labware-creator/components/__tests__/sections/WellBottomAndDepth.test.tsx:95]: I fixed the decision log miswording

**Review by smb2268 — APPROVED** [review]

**Pull request merged**